samples/openthread/cli/README.rst

```diff
@@ -85,11 +85,10 @@ The following configuration files are available:
 
 * :file:`overlay-usb.conf` - Enables USB transport support.
   Additionally, you need to set :makevar:`DTC_OVERLAY_FILE` to :file:`usb.overlay`.
-* :file:`overlay-logging.conf` - Sets more detailed logging.
-* :file:`overlay-rtt.conf` - Redirects logs to RTT.
-  For more information about RTT please refer to :ref:`RTT logging <ug_logging>`.
+* :file:`overlay-logging.conf` - Enables logging (by default redirected to RTT   -
+  for more information about RTT please refer to :ref:`RTT logging <ug_logging>`).
 * :file:`overlay-debug.conf` - Enables debugging the Thread sample with GDB thread awareness.
-* :file:`overlay-ci.conf` - Disables boot banner and shell prompt.
+* :file:`overlay-cert.conf` - Disables boot banner and shell prompt.
```

Comment:
Notice `overlay-ci.conf` is being used in `sample.yaml` and in `ug_thread_certification.rst`. I'd keep that name since `overlay-cert.conf` was used only for Thread Certification purposes.

Reply:
`overlay-cert.conf` is still documented here.